src/NumericalAlgorithms/Interpolation/Callbacks/Callbacks.hpp

```diff
@@ -8,5 +8,5 @@ namespace intrp {
  * \ingroup NumericalAlgorithmsGroup
  * \brief Contains callback functions called by `InterpolationTarget`s.
  */
-namespace callbacks;
+namespace callbacks {}
```

Comment:
(not a change request) please explain the reason for this change, so I understand?

Reply:
Apparently doxygen only documents defined namespaces, not declared namespaces.  Not sure why.  But this change was necessary for `callbacks` documentation to show up in the Namespaces tab in the doxygen dox.